### DISCUSSION TITLE
Add Default Dynamic Variables

### DISCUSSION THREAD

**Discussion by micahjscheer** [directus/directus#15799] [Feature Requests]
I think the ability to use Dynamic Variables in Default Values would be incredibly helpful (it is critical for our business).

Currently setting the Data Model Schema with this;
<img width="854" alt="Screen Shot 2022-09-29 at 3 43 35 pm" src="https://user-images.githubusercontent.com/26317302/192954723-cccb5ec4-47dd-40a2-ae15-1f2f7e2fca74.png">

Results in a form with this;
<img width="844" alt="Screen Shot 2022-09-29 at 3 54 39 pm" src="https://user-images.githubusercontent.com/26317302/192954901-658cdf93-f16a-4bef-b0ac-d5a265175b9a.png">

I think it would be better to result in this;
<img width="844" alt="Screen Shot 2022-09-29 at 3 56 22 pm" src="https://user-images.githubusercontent.com/26317302/192955206-1e7b91d5-fee4-40b5-9590-f640f02326aa.png">

I know you can add Dynamic Variable Defaults into the role permissions but that doesn't work for the Admin role.

**Comment by JLKM** [comment]
Being able to refer and fetch "parent"-fields and make them default values on "child"-record would also be a big leap forward.

Example:
- Parent collection: Creative Works - including type field with choices a la: [article, book, podcast, etc.]
- Join collection: Relations - M2M
- Child collection: Persons - including type field with choices a la: [article, book, podcast, etc.]

After having saved a Creative Work record:

1. Stay on Creative Work and click New Person Relation
2. Type should be inherited from Creative Work and set as the default value in type field on junction collection
3. Lookup from junction table to relevant person would then be filtered down to only persons having the same type - `Type from Creative Work` Equals `Type from Person`. This dynamic filtering is possible, if interface filter is set with moustaches in raw value.

The type field is not ideal, because it's content is redundant. But it brings forward the Creative Work-type, so that it's available for person-filtering.

The clean solution would be a filter with direct reference back to the Creative Work - like `$this.creative_work.type`, but `$this` - probably as well as dot-referencing - apparently is not possible in Directus as of now.

More info:
[#11905](https://github.com/directus/directus/discussions/11905)
[#10986](https://github.com/directus/directus/discussions/10986)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
